src/screens/ProfileScreen/index.js

```diff
@@ -348,7 +363,7 @@ const ProfileScreen = ({route}) => {
   const goToFollowers = (userId, username) => {
     navigation.navigate('Followers', {
       screen: 'TabFollowing',
-      params: {user_id: userId, username, isFollower: true}
+      params: {user_id: userId, username, isFollower: true, initialData: initialFollowerData}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `initialFollowerData` state is passed as a parameter when navigating to the 'Followers' screen. Ensure that this data is used appropriately on the 'Followers' screen.
```diff
+  params: {user_id: userId, username, isFollower: true, initialData: initialFollowerData}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->